src/annotator/sidebar.js

```diff
@@ -221,6 +221,14 @@ export default class Sidebar extends Guest {
     this.crossframe.on('showSidebar', () => this.show());
     this.crossframe.on('hideSidebar', () => this.hide());
 
+    // Re-publish the crossframe event so that anything extending Delegator
```

Comment:
Getting the dance of events and messaging right was a bit of a...thing. It's still not quite where I'd like it, but given that behavior isn't set in stone yet, I don't want to bang my head on the wall too hard. Events/messaging on the annotator side need a refactor at some point, as we already know.